packages/api/src/record.ts

```diff
@@ -576,6 +579,16 @@ export class Record implements RecordModel {
     return str;
   }
 
+  /**
+   * Returns a pagination cursor for the current record given a sort order.
+   *
+   * @param sort the sort order to use for the pagination cursor.
+   * @returns A promise that resolves to a pagination cursor for the current record.
+   */
+  async paginationCursor(sort: DwnDateSort): Promise<DwnPaginationCursor> {
+    return getPaginationCursor(this.rawMessage, sort);
```

Comment:
I've noticed there's this leaky abstraction in some of the monorepo. Currently, the agent cannot be used independently of the api, which leads me to question the necessity of duplicating methods across both.

The  paginationCursor method in api here merely redirects to getPaginationCursor in agent, without adding any additional logic. I think we should start reducing instances of this and fix the leaky abstractions proactively rather than letting them fester. What do you think?

Reply:
Mostly same comments as above, however I'm not sure about something.

> Currently, the agent cannot be used independently of the api.

Can you elaborate on that? It should be the other way around, the `api` cannot be used independently of `agent`.

Reply:
We will revisit after some back and forth with the team